lib/application.js

```diff
@@ -378,11 +379,18 @@ class Application extends api.events.EventEmitter {
   //   callback <Function>
   //     err <Error> | <null>
   updateSchemas({ version }, callback) {
+    if (this.latestSchema && version < this.latestSchema.version) {
```

Comment:
I think there should also be check if `version === this.latestSchema.version` so it won't load the same schema again.